lib/mail/message.rb

```diff
@@ -285,7 +285,7 @@ def reply(*args, &block)
           bracketed_message_id = "<#{message_id}>"
           reply.in_reply_to = bracketed_message_id
           if !references.nil?
-            refs = [references].flatten.map { |r| "<#{r}>" }
```

Comment:
`.flat_map { }` equals to `.map { }.flatten`

`.flatten.map { }` is something different, it can't be changed this way